tests/test_mpi.py

```diff
@@ -1357,7 +1534,7 @@ def test_merge_haloupdate_if_diff_locindices_v1(self, mode):
         op = Operator(eqns)
 
         calls = FindNodes(Call).visit(op)
-        assert len(calls) == 1
+        assert len(calls) == 2
```

Comment:
you should update the test docstring
you should also ensure that one of these two halo_updates is outside the time loop

Reply:
memo

Reply:
Updated